0.2-changes.md

```diff
@@ -0,0 +1,85 @@
+# 0.2 Roadmap
```

Comment:
I thought you hated documentation; this is fantastic!

Reply:
lol i've been trying to keep it up to date with my plan as i've been working on it. ill probably delete it before merging. i figured it doesn't hurt to make the plans public and in sync with commits

Reply:
also the docs i hate writing are more the quickstart guides that are more marketing copy than technical documentation